src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -237,20 +237,19 @@ static int _PaPulseAudio_ProcessAudio(PaPulseAudio_Stream *stream,
         }
     }
 
-    /*
-     * When stopped we should stop feeding or recording right away
+    /* If we have input which is mono and
+     * output which is stereo. We have to copy
+     * mono to monomono which is stereo.
+     * Then just read half and copy
      */
-    if( stream->isStopped && stream->pulseaudioIsStopped )
+    if( isOutputCb &&
+        stream->outputSampleSpec.channels == 2 &&
+        stream->inputSampleSpec.channels == 1)
     {
-        return paStreamIsStopped;
+        pulseaudioInputBytes /= 2;
     }
 
-    /*
-     * This can be called before we have reached out
-     * starting Portaudio stream or Portaudio stream
-     * is stopped
-     */
-    if( !stream->isActive && stream->pulseaudioIsActive )
+    if( !stream->isActive && stream->pulseaudioIsActive && stream->outputStream)
     {
         if(stream->outputStream)
```

Comment:
This `if(stream-outputStream)` is now redundant due to the new `&& stream->outputStream` in the enclosing if statement above. Could be removed, but is purely cosmetic.

Reply:
Ok I've fixed this. Thank you for spotting.